### PR TITLE
Get ansible automation list

### PR DESCRIPTION
**Pull request by ChunxiAlexLuo**
Signed-off-by: Chunxi Luo chuluo@redhat.com

### CODE REVIEW

File: src/v1/schema/ansible.js [hunk 1]
```diff
@@ -5,6 +5,12 @@ import { gql } from 'apollo-server-express';
 
 export const typeDef = gql`
 # define ansible job template
+type AnsibleAutomation {
+  kind: String
+  apiVersion: String
+  metadata: JSON
```
[the remark this answers is not in the document]
Comment (ycao56) [reply src/v1/schema/ansible.js:11]: see https://github.com/open-cluster-management/grc-ui-api/blob/main/src/v1/schema/query.js#L79

**Comment by ycao56** [comment]
@ChunxiAlexLuo please sign off

**Comment by ChunxiAlexLuo** [comment]
Signed-off-by: Chunxi Luo chuluo@redhat.com

File: src/v1/models/ansible.js [hunk 2]
```diff
@@ -4,8 +4,34 @@
...
+      kind: _.get(ans, 'kind', ''),
+      apiVersion: _.get(ans, 'apiVersion', ''),
+      metadata: {
+        name: _.get(ans, 'metadata.name', ''),
+        namespace: _.get(ans, 'metadata.namespace', ''),
+      },
+      spec: {
+        policyRef: _.get(ans, 'spec.policyRef', ''),
+        eventHook: _.get(ans, 'spec.eventHook', ''),
+        mode: _.get(ans, 'spec.mode', ''),
```
Comment (ycao56) [inline src/v1/models/ansible.js:24]: This should be automationDef

**Review by ycao56 — APPROVED** [review]

**Pull request merged**